disperser/apiserver/metrics_v2.go

```diff
@@ -133,20 +136,49 @@ func newAPIServerV2Metrics(registry *prometheus.Registry, metricsConfig disperse
 		[]string{"reason"},
 	)
 
+	dispersalTimestampDrift := promauto.With(registry).NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: namespace,
+			Name:      "dispersal_timestamp_drift_seconds",
+			Help:      "Distribution of timestamp drift for dispersal requests. Negative values indicate timestamps in the future, positive values indicate timestamps in the past.",
+			Buckets:   []float64{-60, -30, -10, -5, -1, -0.5, 0, 0.5, 1, 2, 5, 10, 30, 60, 120, 300},
+		},
+		[]string{"status", "account_id"}, // "accepted" or "rejected", account address
```

Comment:
Account id going to be high cardinality assuming we're tracking arbitrary customers. Maybe it's fine for now though?